asabiris/handlers/webhandler.py

```diff
@@ -147,9 +147,16 @@ async def send_slack(self, request, *, json_data):
 
 		try:
 			await self.App.SendSlackOrchestrator.send_to_slack(json_data)
-
 		except jinja2.exceptions.UndefinedError as e:
```

Comment:
you should catch this error where you apply jinja params. 
when sending the error message on BE, provide at least the file name (location in the library) that is not right. Tell the user what they should do to fix the situation. I.e. PathError means the template is not in the proper place of the Library, right? Provide some hints in the response how to fix the problem.